shell/common/engine.cc

```diff
@@ -291,6 +291,8 @@ bool Engine::HandleLifecyclePlatformMessage(blink::PlatformMessage* message) {
   if (state == "AppLifecycleState.resumed" && have_surface_) {
     ScheduleFrame();
   }
+  runtime_controller_->SetLifecycleState(state);
+  // We always want pass the message forwards directly into the framework.
```

Comment:
Would like to clarify this comment.  How about something like `Always forward these messages to the framework`?